.github/workflows/tests.yaml

```diff
@@ -145,10 +145,31 @@ jobs:
         run: |
           flux check
           flux get all
+      - name: Teardown Flux
+        run: |
+          flux uninstall -s
+      - name: Restore Flux with Terraform
+        run: |
+          export TF_CLI_CONFIG_FILE="${PWD}/.terraformrc"
+          cd examples/github-via-ssh
+          terraform apply -auto-approve -var "github_token=${GITHUB_TOKEN}" -var "github_org=fluxcd-testing" -var "github_repository=${{ steps.vars.outputs.test_repo_name }}"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITPROVIDER_BOT_TOKEN }}
+      - name: Health check Flux
+        run: flux check
+      - name: Change branch with Terraform
+        run: |
+          export TF_CLI_CONFIG_FILE="${PWD}/.terraformrc"
+          cd examples/github-via-ssh
+          terraform apply -auto-approve -var "github_token=${GITHUB_TOKEN}" -var "github_org=fluxcd-testing" -var "github_org=test-branch" -var "github_repository=${{ steps.vars.outputs.test_repo_name }}"
```

Comment:
You've specified `github_org` twice here. The second one should be `git_branch`.